skbio/util/_decorator.py

```diff
@@ -82,3 +85,207 @@ def __get__(self, cls, owner):
 
     def __set__(self, obj, value):
         raise AttributeError("can't set attribute")
+
+
+class _state_decorator(object):
+    """ Base class for decorators of all public functionality.
+    """
+
+    _line_prefix = '\n        '
```

Comment:
Skimmed over this before, but it was a bit of a magic number. Thanks for catching, fixed now.